Storage/src/StorageObject.php

```diff
@@ -620,6 +632,21 @@ public function downloadToFile($path, array $options = [])
      * echo $stream->getContents();
      * ```
      *
+     * ```
+     * // Set the Range header in order to download a subrange (the first 5 bytes)
+     * // of the object.
```

Comment:
Please state that GCS respects spec for [Range](https://tools.ietf.org/html/rfc7233#section-3.1).

It's self-explanatory, but it would also help to add the three cases:
```
// Example values:
// First 3 bytes of an object
'Range' => 'bytes=0-3`
// Start from to end of an object
'Range' => 'bytes=10-'
// Last 3 bytes of an object
'Range' => 'bytes=-3'
```

Reply:
Good call :), added. PTAL when you have a moment.